clang/test/CXX/temp/temp.decls/temp.fct/temp.func.order/p2.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -std=c++20 %s
+// expected-no-diagnostics
+
+template<typename T>
```

Comment:
Nitpick wrap this in `namespace GH77377` since this is a regression test from a bug report